app/views/events/_form.html.erb

```diff
@@ -33,6 +33,8 @@
     <div class="col-lg-10">
       <%= f.text_field :max_participants, :class => 'form-control' %>
       <%=f.error_span(:max_participants) %>
+      <%= f.label :participants_are_unlimited %>
```

Comment:
You could use 0, since 0 does not have a meaning yet. I'm not really happy with either of the solutions